monitor/dashboard.py

```diff
@@ -99,26 +99,30 @@ def supply_information():
             supply_information = dict()
 
             total_supply = economics.total_supply
-            initial_supply = economics.initial_supply
+            initial_supply = 1000000000
+            current_total_supply = economics.initial_supply
+            worklock_supply = economics.worklock_supply
 
             # Overview
-            supply_information['total_supply'] = str(round(NU.from_nunits(total_supply), 2))
-            supply_information['initial_supply'] = str(round(NU.from_nunits(initial_supply), 2))
-            supply_information['staking_rewards_supply'] = str(round(NU.from_nunits(total_supply - initial_supply), 2))
+            supply_information['max_supply'] = str(round(NU.from_nunits(total_supply), 2))
+            supply_information['current_total_supply'] = str(round(NU.from_nunits(current_total_supply), 2))
```

Comment:
@mswilkison did you no longer want to include the `initial_supply` value of `1000000000 NU`?

Reply:
We can include it, but I don't know if it's super relevant. It's relevancy get subsumed by current total supply imo